openlibrary/plugins/upstream/code.py

```diff
@@ -99,6 +99,9 @@ class merge_work(delegate.page):
     path = "/works/merge"
 
     def GET(self):
+        user = web.ctx.site.get_user()
+        if not user or (not user.is_admin() and not user.is_librarian()):
```

Comment:
Also assuming in a future commit this becomes only librarians w/ merge access

Reply:
Nope! Updating :+1: